global.json

```diff
@@ -1,3 +1,3 @@
 {
-    "sources": ["src"]
+    "sources": ["src", "test"]
```

Comment:
Ahhh actually `./build test` doesn't locate the test project without this bit. Not entirely sure why that's the case though.